server/parsec/components/user.py

```diff
@@ -489,6 +499,11 @@ async def list_frozen_users(
     ) -> list[UserID] | UserListFrozenUsersBadOutcome:
         raise NotImplementedError
 
+    async def list_organizations(
```

Comment:
Move this in the organization component since it is about retrieving organizations